src/syft/lib/python/collections/ordered_dict.py

```diff
@@ -130,9 +136,15 @@ def update(self, *args, **kwds: Any) -> SyPrimitiveRet:  # type: ignore
         res = super().update(*args, **kwds)
         return PrimitiveFactory.generate_primitive(value=res)
 
-    def values(self) -> SyPrimitiveRet:
-        res = list(super().values())
-        return PrimitiveFactory.generate_primitive(value=res)
+    def values(self, *args: Any, max_len: Optional[int] = None) -> Iterator:  # type: ignore
+        # this is what the super type does and there is a test in dict_test.py
+        # test_values which checks for this so we could disable the test or
+        # keep this workaround
+        if len(args) > 0:
+            traceback_and_raise(
+                TypeError("values() takes 1 positional argument but 2 were given")
```

Comment:
Could we replace 2 with `len(args)`?

Reply:
sure. I just took this code from dict.py